src/shared/services/tabs-util.service.ts

```diff
@@ -70,6 +71,17 @@ export class TabsUtilService {
     return keyToTabName;
   }
 
+  getTabNameToKeys(keys: Set<string>, schemaKeyToTabName: { [key: string]: string } , tabNames: Array<string>):
```

Comment:
no `space` after before `comma`, I thought that was enforced by `tslint` but apparently not.